prowler/providers/aws/services/elasticache/elasticache_service.py

```diff
@@ -75,6 +72,34 @@ def __describe_cache_subnet_groups__(self, regional_client):
                 f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __describe_replication_groups__(self, regional_client):
+        # Redis Clusters
+        logger.info("Elasticache - Describing Replication Groups...")
+        try:
+            for repl_group in regional_client.describe_replication_groups()[
+                "ReplicationGroups"
+            ]:
+                replication_arn = repl_group["ARN"]
```

Comment:
Add a try/except within the for loop pelase.

Reply:
Done